### ISSUE TITLE
typing a message and pressing "send later" doesn't preserve your message

### ISSUE THREAD

**Issue by tmaybe** [slco-2016/clientcomm#405]
If you type a message in the field at the bottom of the message stream and press "send later", you're taken to a form that prompts you for a type of communication, and, if you select "write message", prompts you for a time, but then doesn't preserve your original message, which you have to retype.

**Comment by alsomanya** [comment]
yes! i've noticed this — it's really annoying. I'm going to add it to this sprint, would love to fix it.
